cmd/tsbs_load_mongo/main.go

```diff
@@ -85,9 +85,14 @@ func init() {
 	metaFieldIndex = viper.GetString("meta-field-index")
 	granularity = viper.GetString("granularity")
 	
-	config.HashWorkers = true
+	if documentPer  {
```

Comment:
Based on my reading of your description:

> To fix this, we simply need to make it so that when we have documentPer set, we only set HashWorkers to true when --batch-meta-fields is true.

Wouldn't you combine the conditions here?
```suggestion
	if documentPer && batchMetaFields {
```

Reply:
No, if documentPer is true, then we by default set HashWorkers to false, and then [later on](https://github.com/mongodb-forks/tsbs/pull/10/files#diff-06075baafe5b4406a965119455ad60cad070e4a3e634f5a8bcc1b62f771bd5faR95), we set HashWorkers true if we have that batchMetaFields is true.

Reply:
Oh, `HashWorkers` is flipped from `documentPer`. Got it now!